src/main.rs

```diff
@@ -25,6 +25,12 @@ fn main() {
         .help("The status code of the previously run command")
         .takes_value(true);
 
+    let pipestatus_arg = Arg::with_name("pipestatus")
+        .long("pipestatus")
+        .value_name("PIPESTATUS")
+        .help("Status codes from the previous pipeline")
```

Comment:
Recommend adding a `long_help` here that documents the specific argument we're looking for. Is it a single, space-separated argument? Several individual arguments? Something else? If possible, I'd consider using [Arg::multiple](https://docs.rs/clap/2.33.0/clap/struct.Arg.html#method.multiple), which means that the expected format would be: `--pipestatus 0 1 0 2` (as opposed to `--pipestatus "0 1 0 2"`.

Reply:
Ah, that’s cool, I’m not really familiar with clap so I wasn’t aware this was a thing, thank you very much!